react-native-libraries.json

```
@@ -10274,5 +10274,14 @@
     "ios": true,
     "android": true,
     "web": true
+  },
+  {
+    "githubUrl": "https://github.com/neurio/react-native-local-network-permission",
+    "npmPkg": "@generac/react-native-local-network-permission",
+    "examples": [
+      "https://github.com/neurio/react-native-local-network-permission/tree/main/example"
+    ],
+    "ios": true,
+    "expo": true
```

Hello @benjaminkomen, thank you for adding new entry to the directory! 👌 

```suggestion
    "ios": true
```

Just to confirm, [`expo` flag](https://github.com/react-native-community/directory?tab=readme-ov-file#expo) means that package will work with Expo Go client, and after taking a quick look at the code it looks like it won't, due to native code included in the package, but feel free to correct me if I have missed something.

---

Ah sorry, I thought it meant the package works with Expo. I will remove it